_reports/2019-2020-finance.md

```diff
@@ -0,0 +1,277 @@
+
+Due to the unprecedented nature of the COVID-19 outbreak, the CS Department instituted a “roll-over” policy for outstanding organization balances. Given that Teach LA did not incur any expenses during Spring Quarter as a result of the switch to virtual instruction during the COVID-19 outbreak, the net difference of CS Department funding (**$429.48**) will be applied to the organization's 2020-2021 balance.
+
+### USAC Community Service Mini Fund
+
+Teach LA also received funding from the CS Mini Fund during the 2019-2020 school year.
+
+The CS Mini Fund is often used as a supplemental funding source for outreach organizations and typically agrees to cover costs associated with transportation, food, additional supplies, and more.
+
+The majority of club expenses for which the CS Mini Fund donation was utilized related to outreach directly.
+
+## Raw Data
+
+[Download the data here.]({{site.baseurl}}/accountability/budget-19-20.json)
+
+<details markdown="0">
+    <summary>Open the table (click/tap here)</summary>
+    <table class="budget-table">
+        <tr>
+            <th>
+                Date
+            </th>
+            <th>
+                Event
+            </th>
+            <th>
+                Type
+            </th>
+            <th>
+                Qty.
+            </th>
+            <th>
+                Item
+            </th>
+            <th>
+                Unit Cost
+            </th>
+            <th>
+                Total Cost
+            </th>
+            <th>
+                Funding Source
+            </th>
+            <th>
+                Special Notes
+            </th>
+        </tr>
+        {% for row in site.data.budget-19-20 %}
+        <tr>
+            <td>
+                {{row.Date}}
+            </td>
+            <td>
+                {{row.Event}}
+            </td>
+            <td>
+                {{row.Type}}
+            </td>
+            <td>
+                {{row.Item}}
+            </td>
+            <td>
+                {{row.Qty}}
+            </td>
+            <td>
+                {{row.Unit}}
+            </td>
+            <td>
+                {{row.Total}}
+            </td>
+            <td>
+                {{row.Source}}
+            </td>
+            <td>
+                {{row.Notes}}
+            </td>
+        </tr>
+        {% endfor %}
+    </table>
+</details>
+
+## Conclusion
+
+This concludes all publishable details about ACM Teach LA's financials. If there are any questions or concerns about the data provided, we respond to emails at [acmteachla@gmail.com](mailto:acmteachla@gmail.com).
+
+We plan on continuing this trend of financial transparency in the 2020-2021 academic year, even as the majority of our activities remain online. We are committed to transparency as a core tenet of our organization.
+
+<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
```

Comment:
Are we thinking that these plots and breakdowns will be done on a per-report basis?

Reply:
Yup! Any thoughts on different approaches here?

Reply:
I was just thinking it might be wise to include a specific accountability layout or header to keep these imports out of the way of the report contents. The charts should probably still be generated within the document, though, since we might want to highlight specific plots.

Reply:
We can punt this to another PR, also depends if we keep the same graphing libraries and stuff.